third_party/docfx-doclet-143274/src/main/java/com/microsoft/model/ApiVersionPackageToc.java

```diff
@@ -0,0 +1,116 @@
+package com.microsoft.model;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.LinkedHashMap;
+import java.util.List;
+
+public class ApiVersionPackageToc {
+  static final String CLIENTS = "Clients";
+  static final String REQUESTS_AND_RESPONSES = "Requests and responses";
+  static final String SETTINGS = "Settings";
+  static final String ALL_OTHERS = "All other classes and interfaces";
+  static final String BUILDERS = "Builders";
+  static final String ENUMS = "Enums";
+  static final String INTERFACES = "Interfaces";
+  static final String MESSAGES = "Messages";
+  static final String EXCEPTIONS = "Exceptions";
+  static final String PAGING = "Paging";
```

Comment:
(please correct me if I'm wrong) I think `Paging` methods would be under a `Client`'s page in Cloud RAD, right? So to reorganize it I think it would a righthand navigation change rather than lefthand (which we've decided is not feasible currently).

I think the same is true with `Resource name`.

Reply:
Paging methods are part of the client, but the paging classes themselves and paging-specific requests+responses are their own entries in the left-hand nav. Ex: https://cloud.google.com/java/docs/reference/google-cloud-logging/latest/com.google.cloud.logging.v2.ConfigClient contains a bunch of paging methods, but there are dedicated RAD pages for:
* https://cloud.google.com/java/docs/reference/google-cloud-logging/latest/com.google.cloud.logging.v2.ConfigClient.ListBucketsPage
* https://cloud.google.com/java/docs/reference/google-cloud-logging/latest/com.google.cloud.logging.v2.ConfigClient.ListBucketsPagedResponse
* https://cloud.google.com/java/docs/reference/google-cloud-logging/latest/com.google.cloud.logging.v2.ConfigClient.ListExclusionsPage
* https://cloud.google.com/java/docs/reference/google-cloud-logging/latest/com.google.cloud.logging.v2.ConfigClient.ListExclusionsPagedResponse
* ... and more.

Nothing is changing here about the client's right-hand navigation organization showing its methods. This is maintaining but reorganizing the left-hand entries.